packages/datetime/test/dateRangePickerTests.tsx

```diff
@@ -57,43 +57,45 @@ describe("<DateRangePicker>", () => {
             const minDate = new Date(2000, Months.JANUARY);
             renderDateRangePicker({ defaultValue, initialMonth, maxDate, minDate });
-            assert.equal(dateRangePicker.state.displayYear, 2002);
-            assert.equal(dateRangePicker.state.displayMonth, Months.MARCH);
+            assert.equal(dateRangePicker.state.leftView.getYear(), 2002);
+            assert.equal(dateRangePicker.state.leftView.getMonth(), Months.MARCH);
         });
 
         it("is defaultValue if set and initialMonth not set", () => {
             const defaultValue = [new Date(2007, Months.APRIL, 4), null] as DateRange;
             const maxDate = new Date(2020, Months.JANUARY);
             const minDate = new Date(2000, Months.JANUARY);
             renderDateRangePicker({ defaultValue, maxDate, minDate });
-            assert.equal(dateRangePicker.state.displayYear, 2007);
-            assert.equal(dateRangePicker.state.displayMonth, Months.APRIL);
+            assert.equal(dateRangePicker.state.leftView.getYear(), 2007);
+            assert.equal(dateRangePicker.state.leftView.getMonth(), Months.APRIL);
         });
 
         it("is value if set and initialMonth not set", () => {
             const maxDate = new Date(2020, Months.JANUARY);
             const minDate = new Date(2000, Months.JANUARY);
             const value = [new Date(2007, Months.APRIL, 4), null] as DateRange;
             renderDateRangePicker({ maxDate, minDate, value });
-            assert.equal(dateRangePicker.state.displayYear, 2007);
-            assert.equal(dateRangePicker.state.displayMonth, Months.APRIL);
+            assert.equal(dateRangePicker.state.leftView.getYear(), 2007);
+            assert.equal(dateRangePicker.state.leftView.getMonth(), Months.APRIL);
         });
 
         it("is today if only maxDate/minDate set and today is in date range", () => {
             const maxDate = new Date(2020, Months.JANUARY);
             const minDate = new Date(2000, Months.JANUARY);
             const today = new Date();
             renderDateRangePicker({ maxDate, minDate});
-            assert.equal(dateRangePicker.state.displayYear, today.getFullYear());
-            assert.equal(dateRangePicker.state.displayMonth, today.getMonth());
+            assert.equal(dateRangePicker.state.leftView.getYear(), today.getFullYear());
+            assert.equal(dateRangePicker.state.leftView.getMonth(), today.getMonth());
         });
 
         it("is a day between minDate and maxDate if only maxDate/minDate set and today is not in range", () => {
             const maxDate = new Date(2005, Months.JANUARY);
             const minDate = new Date(2000, Months.JANUARY);
             renderDateRangePicker({ maxDate, minDate });
-            const { displayMonth, displayYear } = dateRangePicker.state;
-            assert.isTrue(DateUtils.isDayInRange(new Date(displayYear, displayMonth), [minDate, maxDate]));
+            const leftDisplayMonth = dateRangePicker.state.leftView;
```

Comment:
nit: Pondering to myself it `const leftView = ...` would be clearer to me than `const leftDisplayMonth = ...`, since the display month includes a year. Thoughts @leebyp?